site/src/routes/nfts-check.js

```diff
@@ -0,0 +1,34 @@
+import { JSONResponse } from '../utils/json-response.js'
+import { get as getDeals } from '../models/deals.js'
+import * as cluster from '../cluster.js'
+import { HTTPError } from '../errors.js'
+
+/**
+ * @typedef {import('../bindings').Deal} Deal
+ */
+
+/**
+ * @param {FetchEvent} event
+ * @param {Record<string,string>} params
+ */
+export const check = async (event, params) => {
+  const { cid } = params
+
+  /** @type {import('@nftstorage/ipfs-cluster').StatusResponse} */
+  let status
+  /** @type {Deal[]} */
+  let deals
+  try {
+    ;[status, deals] = await Promise.all([cluster.status(cid), getDeals(cid)])
+  } catch (err) {
```

Comment:
Aside: seems like something router should be doing this.

Reply:
> I do not see a reason why we could not expose `size` without auth & I'm not entirely sure I see a reason why `created` timestamp needs to be a secret either. And if both are true we could probably make `client.status` not require `auth`. It would probably need switch to use this implementation instead, but smaller API surface seems like a win.

Just surfacing our convo:

We respond with more data in the HTTP API than we return from the JS client https://github.com/ipfs-shipyard/nft.storage/blob/4dfcb5b3537c0a7f3628d70c8ca97c7cda2d93f3/website/public/schema.yml#L147-L174 

for example `scope` is the name of the token and can be considered private.

Also, for things pinned through the pinning service API it'll carry `name` and `meta` in the `pin` object which is user specific.

So it would be a breaking change to remove that data.

In a way it's good because we can support the concept of "user private" data in the "status API" and have public stuff in the "check API"...